foundation/docs/foundation/about.md

```diff
@@ -5,4 +5,16 @@ title: About Us
 summary: Who we are
 ---
 # About Us
-The PyTexas Foundation is a 501(c)3 non-profit run by a Texas-based volunteer group. We are Python enthusiasts that want to share the programming language with the world, starting right here in Texas. The PyTexas Conference is currently the foundation’s only fundraising event of the year.
\ No newline at end of file
+The PyTexas Foundation is a 501(c)3 non-profit run by a Texas-based volunteer 
+group. We are Python enthusiasts that want to share the programming language 
+with the world, starting right here in Texas. We host and support various programs
+that benefit the Python Community in Texas. We host an annual conference and 
+monthly meetup, maintain a virtual community for the Python developers in Texas
+and beyond, and support various meetups in cities throughout the state.
```

Comment:
Maybe change `various programs` to `a variety of` or eliminate the repeated `various` in `support various meetups` to `support meetups`

Reply:
Done